wp-content/themes/pub/wporg-learn-2024/src/card-course/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Adds aria labels to the blocks in the card course template.
+ */
+
+namespace WordPressdotorg\Theme\Learn_2024\Card_Course;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Actions and filters.
+ */
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Add the scripts to update the blocks in the card course template.
+ *
+ * The dependencies are autogenerated in block.json, and can be read with
+ * `wp_json_file_decode` & `register_block_script_handle.
+ */
+function init() {
+	$metadata_file = dirname( dirname( __DIR__ ) ) . '/build/card-course/block.json';
+	$metadata = wp_json_file_decode( $metadata_file, array( 'associative' => true ) );
+	$metadata['file'] = $metadata_file;
+
+	$script_handle = register_block_script_handle( $metadata, 'viewScript', 0 );
+
+	// Enqueue the assets when the card course template is on the page.
+	add_action(
+		'render_block_core/template-part',
+		function( $block_content, $block ) use ( $script_handle ) {
+			$slugs = array( 'card-course-h3', 'card-course', 'card-lesson-h3', 'card-lesson', 'card' );
```

Comment:
I think we need `card-lesson-plan` too

Reply:
I don't see the card-lesson-plan, and the lesson plan archive is already working now. It doesn't work on your side?

Reply:
Oh yeah I was looking at sandbox, I must have synced a file in there at some point, ignore